.htmlvalidate.json

```diff
@@ -6,6 +6,7 @@
     "html-validate:recommended"
   ],
   "rules": {
-    "void-style": "off"
+    "void-style": "off",
+    "prefer-button": "warn"
```

Comment:
Let's put this in a different PR?

Reply:
Ah it is merged. NVM

Reply:
Noted, will do next time. I was trying to make sure I didn't push something with a failed build.